packages/bvaughn-architecture-demo/components/Icon.tsx

```diff
@@ -47,12 +49,20 @@ export default function Icon({
       path =
         "M21.99 4c0-1.1-.89-2-1.99-2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4-.01-18zM18 14H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z";
       break;
+    case "comments":
+      path =
+        "M18.0487 4.01083H17.046V12.0325C17.046 12.584 16.5948 13.0352 16.0433 13.0352H4.01083V14.0379C4.01083 15.1409 4.91327 16.0433 6.01625 16.0433H16.0433L20.0542 20.0542V6.01625C20.0542 4.91327 19.1517 4.01083 18.0487 4.01083ZM15.0406 9.02437V2.00542C15.0406 0.902437 14.1382 0 13.0352 0H2.00542C0.902437 0 0 0.902437 0 2.00542V15.0406L4.01083 11.0298H13.0352C14.1382 11.0298 15.0406 10.1274 15.0406 9.02437Z";
+      break;
     case "confirm":
       path = "M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z";
       break;
     case "delete":
       path = "M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z";
       break;
+    case "document":
+      path =
+        "M8.8489 0.492998C8.53137 0.175474 8.10522 0 7.66236 0H1.67118C0.752031 0 0 0.752031 0 1.67118V15.0406C0 15.9598 0.743675 16.7118 1.66282 16.7118H11.6983C12.6174 16.7118 13.3694 15.9598 13.3694 15.0406V5.70708C13.3694 5.26422 13.194 4.83807 12.8764 4.5289L8.8489 0.492998ZM9.19149 13.3694H4.17795C3.71838 13.3694 3.34236 12.9934 3.34236 12.5339C3.34236 12.0743 3.71838 11.6983 4.17795 11.6983H9.19149C9.65106 11.6983 10.0271 12.0743 10.0271 12.5339C10.0271 12.9934 9.65106 13.3694 9.19149 13.3694ZM9.19149 10.0271H4.17795C3.71838 10.0271 3.34236 9.65106 3.34236 9.19149C3.34236 8.73192 3.71838 8.3559 4.17795 8.3559H9.19149C9.65106 8.3559 10.0271 8.73192 10.0271 9.19149C10.0271 9.65106 9.65106 10.0271 9.19149 10.0271ZM7.52031 5.01354V1.25339L12.1161 5.84913H8.3559C7.89632 5.84913 7.52031 5.47311 7.52031 5.01354Z";
```

Comment:
Where'd this one come from? It doesn't resize properly.

Going to swap it with one from Material's free icon set:
```js
path = "M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6zm2 16H8v-2h8v2zm0-4H8v-2h8v2zm-3-5V3.5L18.5 9H13z"
```

Reply:
Before:
![Screen Shot 2022-07-05 at 8 04 12 AM](https://user-images.githubusercontent.com/29597/177323307-d73e8595-2a88-45e1-b8e5-cb2c685aa1ad.png)

After:
![Screen Shot 2022-07-05 at 8 04 17 AM](https://user-images.githubusercontent.com/29597/177323296-b0feb913-aeb0-4ea4-a4f2-7b921cf1e5c6.png)